features/employment/statutory_pay.feature

```diff
@@ -3,111 +3,39 @@ Feature:
-            | ccms_code | client_involvement_type |
-            | SE013     | A                       |
-        And I add the following statutory sick pay details for the client:
-            | client_id |     date     |  gross | benefits_in_kind  | tax   | national_insurance | net_employment_income  |
-            |     C     |  2022-07-22  | 500.50 |       0           | 0.00 |       0.0           |        500.50          |
-            |     C     |  2022-08-22  | 500.50 |       0           | 0.00 |       0.0           |        500.50          |
-            |     C     |  2022-09-22  | 500.50 |       0           | 0.00 |       0.0           |        500.50          |
+        And The "client" earns 500 per month in statatory sick pay
         When I retrieve the final assessment
         Then I should see the following "employment" details:
             | attribute                  | value    |
             | fixed_employment_deduction | 0.0      |
 
     Scenario: The client is receiving statutory sick pay only but has entered childcare costs
         Given I am undertaking a certificated assessment
-        And A submission date of "2023-07-26"
-        And I add the following applicant details for the current assessment:
-            | date_of_birth               | 1979-12-20 |
-            | involvement_type            | applicant  |
-            | has_partner_opponent        | false      |
-            | receives_qualifying_benefit | false      |
-            | employed                    | true       |
-        And I add the following proceeding types in the current assessment:
-            | ccms_code | client_involvement_type |
-            | SE013     | A                       |
-        And I add the following statutory sick pay details for the client:
-            | client_id |     date     |  gross | benefits_in_kind  | tax   | national_insurance | net_employment_income  |
-            |     C     |  2022-07-22  | 500.50 |       0           | 0.00 |       0.0           |        500.50          |
-            |     C     |  2022-08-22  | 500.50 |       0           | 0.00 |       0.0           |        500.50          |
-            |     C     |  2022-09-22  | 500.50 |       0           | 0.00 |       0.0           |        500.50          |
-        And I add the following outgoing details for "child_care" in the current assessment:
-            | payment_date | client_id | amount |
-            | 2020-02-29   | og-id1    | 200.00 |
-            | 2020-03-27   | og-id2    | 200.00 |
-            | 2020-04-26   | og-id3    | 200.00 |
-        And I add the following dependent details for the current assessment:
-            | date_of_birth | in_full_time_education | relationship   |
-            | 2018-12-20    | FALSE                  | child_relative |
+        And The "client" earns 500 per month in statatory sick pay
+        And I add outgoing details for "child_care" of 200 per month
+        And I have a dependant aged 2
```

Comment:
this is a minor nitpick, does the order matter here? if not I think it makes more sense to say:

And I have a dependant aged 2
And I add outgoing details for "child_care" of 200 per month

rather than adding costs then declaring a child.